source/whats-new.txt

```diff
@@ -79,6 +79,19 @@ improvements, and fixes:
   </reference/map-reduce-to-aggregation-pipeline/>` in the
   {+mdb-server+} manual.
 
+- The following methods return ``void`` instead of the raw command
+  response:
+
+  - ``MongoDB\\Client``: ``dropDatabase()``
+  - ``MongoDB\\Collection``: ``drop()``, ``dropIndex()``,
+    ``dropIndexes()``, ``dropSearchIndex()``, ``rename()``
+  - ``MongoDB\\Database``: ``createCollection()``, ``drop()``,
+    ``dropCollection()``, ``renameCollection()``
```

Comment:
Would it make sense to highlight this as a breaking change so it stands out a bit more? Similar to what we do in the [C++ docs here](https://www.mongodb.com/docs/languages/cpp/cpp-driver/current/whats-new/#what-s-new-in-3.11)?